package.json

```diff
@@ -81,7 +81,7 @@
     "workbox-build": "4.3.1",
     "write-file-webpack-plugin": "^4.5.1"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
I am not sure if it is valid to have 2 devDependencies sections.  Did you uninstall and re-install the "ts-jest" package or did you manually edit the package.json file?  The ts-jest package should just appear inline with the other devDependencies.  Using npm uninstall/install will automatically do that for you.

Reply:
I manually edited and re-ran build-dev/test and seemed to run fine. Let me move it up to be sure. Thanks.